client/my-sites/email/email-management/style.scss

```diff
@@ -219,11 +219,11 @@
 	line-height: 24px;
 
 	> img, > svg {
-		height: 18px;
+		height: 24px;
 		fill: var( --color-neutral-50 );
 		filter: grayscale( 100% );
 		margin-right: 8px;
-		width: 18px;
+		width: 24px;
 	}
```

Comment:
Do we still need the `fill` and `filter` rules since the new images already are in black-and-white?

Reply:
I also wonder if we need to define `height` and `width` - I don't see any visual difference when I disable those rules.

Reply:
> I also wonder if we need to define height and width - I don't see any visual difference when I disable those rules.
Disabling them won't work because there are defaults on the viewBox. Try changing them instead.

> Do we still need the fill and filter rules since the new images already are in black-and-white?
`fill` and `filter` may not be needed @AllTerrainDeveloper. The images are in grayscale already so the transformation isn't noticeable.